test/CTBindCreator.t.sol

```diff
@@ -42,12 +36,20 @@ contract CreatorTechTest is Test {
         assertEq(ctCreatorAddr, creatorAddr);
     }
 
+    function testBindCreatorAndClaim_haveUnclaimedFees() public {}
```

Comment:
nit: What's the plan with these? Perhaps can write some TODOs if that's the plan.

Reply:
fill this